src/NzbDrone.Core/Localization/Core/es.json

```diff
@@ -1,2108 +1,2108 @@
 {
```

Comment:
Are all the formatting changes wanted or is it your formater derping 🤔

Reply:
Formatter went rogue 😞